tests/MongoDB.Driver.Core.TestHelpers/JsonDrivenTests/CommandStartedEventAsserter.cs

```diff
@@ -62,23 +69,43 @@ private void AdaptExpectedUpdateModels(List<BsonDocument> actualModels, List<Bso
                 var actualModel = actualModels[i];
                 var expectedModel = expectedModels[i];
 
-                if (expectedModel.Contains("multi") && expectedModel["multi"] == false && !actualModel.Contains("multi"))
+                if (!actualModel.Contains("multi"))
+                {
+                    actualModel.Add("multi", false);
+                }
+                if (!actualModel.Contains("upsert"))
                 {
-                    expectedModel.Remove("multi");
+                    actualModel.Add("upsert", false);
                 }
 
-                if (expectedModel.Contains("upsert") && expectedModel["upsert"] == false && !actualModel.Contains("upsert"))
+                if (!expectedModel.Contains("multi"))
                 {
-                    expectedModel.Remove("upsert");
+                    expectedModel.Add("multi", false);
                 }
+                if (!expectedModel.Contains("upsert"))
+                {
+                    expectedModel.Add("upsert", false);
+                }
+
+                // todo: check on evergreen
+                //if (expectedModel.Contains("multi") && expectedModel["multi"] == false && !actualModel.Contains("multi"))
+                //{
+                //    expectedModel.Remove("multi");
+                //}
+
+                //if (expectedModel.Contains("upsert") && expectedModel["upsert"] == false && !actualModel.Contains("upsert"))
+                //{
+                //    expectedModel.Remove("upsert");
+                //}
             }
         }
 
         private void AssertCommandAspects(BsonDocument actualCommand, BsonDocument aspects)
         {
-            RecursiveFieldSetter.SetAll(actualCommand, "getMore", 42L);
-            RecursiveFieldSetter.SetAll(actualCommand, "afterClusterTime", 42);
-            RecursiveFieldSetter.SetAll(actualCommand, "recoveryToken", 42);
+            foreach (var template in _templates)
```

Comment:
some tests expected validation of `getMore` value. So, I added the ability to configure the list of templates which should be replaced on `42` template value